pkg/network/network.go

```diff
@@ -294,7 +269,7 @@ func (m *Setup) SetArgs(args []string) error {
 
 				splittedPort := strings.SplitN(value, "/", 2)
 				if len(splittedPort) != 2 {
-					return fmt.Errorf("badly formatted portmap argument '%s', must be of form portmap=hostPort:containerPort/protocol", splitted[1])
+					return fmt.Errorf("badly formatted portmap argument '%s', must be of form portmap=hostPort:containerPort/protocol", splitted)
```

Comment:
Wouldn't it be more clear if you use the original value here (arg)?

Reply:
@mem arg could contain arg not related to portmap, finally changed for `value` to reflect the argument value generating the error

Reply:
Ok